.travis.yml

```diff
@@ -0,0 +1,35 @@
+# Travis yml file inspired by scikit-bio
+# Check on http://lint.travis-ci.org/ after modifying it!
+sudo: false
+language: python
+env:
+  - PYVERSION=3.5 USE_CYTHON=TRUE MAKE_DOC=TRUE
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+  - conda update --yes conda
+  - conda info -a
+install:
+  - wget -q https://raw.githubusercontent.com/qiime2/environment-files/master/latest/staging/qiime2-latest-py36-linux-conda.yml
+  - conda env create -q -n test-env --file qiime2-latest-py36-linux-conda.yml
+  - conda install --yes -n test-env --file ci/conda_requirements.txt -c biocore
+  - conda install --yes -n test-env cython
+  - source activate test-env  
+  - pip install -r ci/pip_requirements.txt
+  - pip install -e .
+  - sudo pip install flake8
```

Comment:
is this consistent with ```sudo=false``` in line 3?

Reply:
no good catch, was just bypassing the sudo in travis